src/Database/Type/DecimalType.php

```diff
@@ -100,10 +101,34 @@ public function toDatabase($value, Driver $driver)
     public function toPHP($value, Driver $driver)
     {
         if ($value === null) {
-            return null;
+            return $value;
+        }
+
+        // Using coersion is faster than casting
+        // @codingStandardsIgnoreStart
+        return (float)+$value;
```

Comment:
I benchmarked this agains `(int)$value` and just `0 + $value`. This version won by a surprisingly big margin.

Reply:
Interesting, I would have expected them to come out the same.